### PR TITLE
Adding clarification as part of #1052

### PR DESCRIPTION
**Pull request by brianjking**

### CODE REVIEW

File: source/getting-started/installation-raspberry-pi-image.markdown [hunk 1]
```diff
@@ -12,7 +12,7 @@ footer: true
 The easiest way to install Home Assistant on your Raspberry Pi is by using Hassbian: a Raspberry Pi image with Home Assistant built-in. The image will install the latest version of Home Assistant on initial boot (~5 minutes).
 
  1. [Download the latest image][image-download]
- 2. Flash the image to an SD card:
+ 2. Flash the image to a Micro SD card:
```
Comment (fabaff) [inline source/getting-started/installation-raspberry-pi-image.markdown]: At least the first version of a Raspberry Pi is using an standard-sized SD card.

File: source/getting-started/installation-raspberry-pi-image.markdown [hunk 2]
```diff
@@ -21,7 +21,7 @@ The easiest way to install Home Assistant on your Raspberry Pi is by using Hassb
 
 These instructions are also available as a [video](https://www.youtube.com/watch?v=iIz6XqDwHEk).
 
-Home Assistant will now be available by navigating with a browser to `http://ip-address-of-pi:8123`. The default username is `pi` and password is `raspberry` (please change this by running `passwd`)
+Home Assistant will now be available by navigating with a browser to `http://ip-address-of-pi:8123`. The default username is `pi` and password is `raspberry` (please change this by running `passwd` via an SSH connection to your Pi.)
```
Comment (fabaff) [inline source/getting-started/installation-raspberry-pi-image.markdown]: I guess you can change the password as we if you using a local shell.

File: source/getting-started/index.markdown [hunk 3]
```diff
@@ -13,7 +13,7 @@ Below you can find the standard instructions. See the sidebar for installation i
 
 [Virtualenv Linux instructions]: /getting-started/installation-virtualenv/
 
-Installing and running Home Assistant on your local machine is easy. Make sure you have [Python 3.4 or higher](https://www.python.org/downloads/) installed and execute the following code in a console:
+Installing and running Home Assistant on your local machine is easy. Make sure you have [Python 3.4 or higher](https://www.python.org/downloads/) and [PIP](https://docs.python.org/3/installing/) installed and execute the following code in a console:
```
Comment (fabaff) [inline source/getting-started/index.markdown]: The project calls itself `pip` (all lower-case).

**Comment by Landrash** [comment]
This one and #1073 covers a lot of the same things?

File: source/getting-started/index.markdown (hunk 3 above)
Comment (Landrash) [inline source/getting-started/index.markdown]: And `pip` is in Python3 part of the python package. Some distributions do package it separately.

**Comment by Landrash** [comment]
Closing this and #1073 
Please consolidate them into one and address the requested changes @brianjking .

**Pull request closed without merging**